### ISSUE TITLE
A hud text wrap proposal

### ISSUE THREAD

**Issue by anyingiit** [Zezombye/overpy#152]
```
rule "test":
    @Event eachPlayer
    hudText(hostPlayer,
    "abc
    test",
    "efg",
    "hij",HudPosition.TOP,0,Color.GREEN,Color.BLUE,Color.ORANGE,HudReeval.SORT_ORDER_AND_STRING,SpecVisibility.DEFAULT)
```
> output
```
rule("test")
{
	event
	{
		Ongoing - Each Player;
		All;
		All;
	}

	actions
	{
		Create HUD Text(Host Player, Custom String("abc\n    test"), Custom String("efg"), Custom String("hij"), Top, 0, Green, Blue,
			Orange, Sort Order and String, Default Visibility);
	}
}
```
![YaiYWT.png](https://s1.ax1x.com/2020/05/13/YaiYWT.png)
This is not what I expected. Changing the code to this is what I expected
```
rule "test":
    @Event eachPlayer
    hudText(hostPlayer,
"abc
test",
    "efg",
    "hij",HudPosition.TOP,0,Color.GREEN,Color.BLUE,Color.ORANGE,HudReeval.SORT_ORDER_AND_STRING,SpecVisibility.DEFAULT)
```
If only I could just use "\n"!

**Comment by Zezombye** [comment]
You should be able to use \n. If it doesn't work, please report that as a bug.

The behavior with tabs in multiline strings is annoying, but consistent with other languages that have multiline strings. As such, I am leaving the current behavior in.

**Comment by Zezombye** [comment]
Fixed the \n not working properly in 5.1.1.
